substrate/frame/identity/src/types.rs

```diff
@@ -253,6 +253,8 @@ pub trait IdentityInformationProvider:
 ///
 /// NOTE: This is stored separately primarily to facilitate the addition of extra fields in a
 /// backwards compatible way through a specialized `Decode` impl.
+//
+// TODO: Perhaps just move `username` into a field of `Registration`?
```

Comment:
Probably good to create an issue for this?

Reply:
I actually removed it, just haven't pushed the commit yet.